tests/test_decoding.py

```diff
@@ -1359,6 +1362,8 @@ def test_min_max_length_type_end_of_pdu_in_structure(self):
 
         req_param1._resolve_odxlinks(odxlinks)
         req_param2._resolve_odxlinks(odxlinks)
+        req_param1._resolve_snrefs(None)
+        req_param2._resolve_snrefs(None)
```

Comment:
Shouldn't calling the  resolve snrefs on the parent structure be enough?

Reply:
it would be. The problem there is that this might require to construct a full `DiagLayer` object. (The code here is a pretty bad hack which takes advantage of the fact that these parameter classes do not actually access the passed `diag_layer` object...)